ext/standard/crypt.c

```diff
@@ -129,12 +129,12 @@ PHPAPI zend_string *php_crypt(const char *password, const int pass_len, const ch
 
 			crypt_res = php_sha512_crypt_r(password, salt, output, PHP_MAX_SALT_LEN);
 			if (!crypt_res) {
-				memset(output, 0, PHP_MAX_SALT_LEN);
+				explicit_bzero(output, PHP_MAX_SALT_LEN);
```

Comment:
As we already define the macro, might be better to use `ZEND_SECURE_ZERO` here to avoid problems on win32.